fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/common/CommonDatabaseAdapter.java

```diff
@@ -514,7 +515,7 @@ public void createSequence(String schemaName, String sequenceName, int cache) {
          */
         // The move to start with 1000 gives room for manual creation and update of sequences.
```

Comment:
This comment seems no longer applicable.

Reply:
updated